proto/vtctldata.proto

```diff
@@ -389,6 +389,17 @@ message GetTabletsRequest {
   string shard = 2;
   // Cells is an optional set of cells to return tablets for.
   repeated string cells = 3;
+  // Strict specifies how the server should treat failures from individual
+  // cells.
+  //
+  // When false (the default), GetTablets will return data from any cells that
+  // return successfully, but will fail the request if all cells fail. When
+  // true, any individual cell can fail the full request.
+  bool strict = 4;
+  // TabletAliases is an optional list of tablet aliases to fetch Tablet objects
+  // for. If specified, Keyspace, Shard, and Cells are ignored, and tablets are
+  // looked up by their respective aliase's Cells directly.
```

Comment:
tiny typo - you can choose to fix it or not :grin:

Reply:
😬  let me push up a fix! lol